aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -131,7 +141,11 @@ Resources:
         # This script below is to install AmazonCloudWatchAgent, restart AmazonCloudWatchAgent and tell the result to cloudformation.
         Fn::Base64: !Sub |
            #!/bin/bash
-           rpm -Uvh https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+           curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+           unzip awscliv2.zip
+           sudo ./aws/install
+           aws s3 cp ${PackageLocation} .
```

Comment:
This is not going to work, is it? The default value is the public HTTPS endpoint that serves the RPM, not the S3 formatted URL

Reply:
Can we make it use  https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm by default. That way the default is the same as before.

Otherwise, does the image have aws CLI installed and configured? I can't remember if you have to run `aws configure` first, or if a credential is required to use AWS CLI. (